packages/commons-ui-next/src/utils/isExternalUrl.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
+
+function isExternalUrl(url) {
+  return (
+    typeof url === "string" &&
+    (url.indexOf("http") === 0 || url.indexOf("mailto:") === 0)
```

Comment:
Just finding something to comment on. Maybe we should support even `tel`?

Reply:
I think we only added mailto to support contact us via email which we don't even use (contact form being a Mailchimp embed).